aiger/parser.py

```diff
@@ -1,9 +1,13 @@
+from collections import namedtuple
```

Comment:
Not sure how Visitor class exactly interacts with the grammar. Are Symbol and SymbolTable only local? If so, they should get an underscore.

Reply:
Yeah, they come a old version of the AIG representation, but ended up being nice for bookkeeping. I'll make them private.